kolibri/core/assets/src/mixins/responsive-window.js

```diff
@@ -147,56 +146,36 @@ function windowMetrics() {
   };
 }
 
-import { throttle } from 'frame-throttle';
-
-const windowResizeHandler = throttle(e => {
-  const metrics = windowMetrics();
-  windowListeners.forEach(cb => cb(metrics));
-});
-
 function addWindowListener(cb) {
-  windowListeners.push(cb);
-  cb(windowMetrics()); // call it once initially
+  const listener = throttle(cb);
+  if (window.addEventListener) {
+    window.addEventListener('resize', listener, true);
+  } else if (window.attachEvent) {
+    window.attachEvent('onresize', listener);
+  }
 }
 
 function removeWindowListener(cb) {
-  windowListeners.pop(cb);
-}
-
-/* setup */
-
-if (window.addEventListener) {
-  window.addEventListener('resize', windowResizeHandler, true);
-} else if (window.attachEvent) {
-  window.attachEvent('onresize', windowResizeHandler);
+  const listener = throttle(cb);
```

Comment:
Hm. I think I misunderstood the documentation, then: https://github.com/pelotoncycle/frame-throttle#use